lens/plotting.py

```diff
@@ -171,6 +171,8 @@ def plot_correlation_mpl(ls, include=None, exclude=None):
 
     ax.xaxis.tick_top()
 
+    # Enforces a width of 2.5 inches per cell in the plot,
+    # unless this exceeds 600 pixels.
```

Comment:
Where did that 600 pixels number come from?

As a side-note, replacing `w` with `width_inches` would be a lot clearer.